### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by fdfdffdf** [PocketMine/PocketMine-MP#4051]
```
Error: Call to a member function setMaxValue() on null
File: /src/pocketmine/entity/Living
Line: 82
Type: notice

Code:
[73] 		if($this->isAlive() and !$wasAlive){
[74] 			$pk = new EntityEventPacket();
[75] 			$pk->eid = $this->getId();
[76] 			$pk->event = EntityEventPacket::RESPAWN;
[77] 			Server::broadcastPacket($this->hasSpawned, $pk);
[78] 		}
[79] 	}
[80] 
[81] 	public function setMaxHealth($amount){
[82] 		$this->attributeMap->getAttribute(Attribute::HEALTH)->setMaxValue($amount);
[83] 	}
[84] 
[85] 	public function saveNBT(){
[86] 		parent::saveNBT();
[87] 		$this->namedtag->Health = new ShortTag("Health", $this->getHealth());
[88] 	}
[89] 
[90] 	public abstract function getName();
[91] 
[92] 	public function hasLineOfSight(Entity $entity){

Backtrace:
#0 /src/pocketmine/entity/Entity__64bit(273): pocketmine\entity\Squid->initEntity(boolean)
#1 /src/pocketmine/entity/Entity__64bit(425): pocketmine\entity\Entity->__construct(pocketmine\level\format\mcregion\Chunk object, pocketmine\nbt\tag\CompoundTag pocketmine\nbt\tag\CompoundTag{.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\DoubleTag:245.5.pocketmine\nbt\tag\DoubleTag:68.pocketmine\nbt\tag\DoubleTag:164.5.}.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\DoubleTag:0.pocketmine\nbt\tag\DoubleTag:-0.pocketmine\nbt\tag\DoubleTag:0.}.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\FloatTag:-0.pocketmine\nbt\tag\FloatTag:-0.}.pocketmine\nbt\tag\FloatTag:0.pocketmine\nbt\tag\ShortTag:0.pocketmine\nbt\tag\ShortTag:300.pocketmine\nbt\tag\ByteTag:1.pocketmine\nbt\tag\ByteTag:0.pocketmine\nbt\tag\ShortTag:5.pocketmine\nbt\tag\StringTag:Squid.})
#2 /src/pocketmine/level/format/generic/BaseFullChunk(148): pocketmine\entity\Entity::createEntity(string Squid, pocketmine\level\format\mcregion\Chunk object, pocketmine\nbt\tag\CompoundTag pocketmine\nbt\tag\CompoundTag{.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\DoubleTag:245.5.pocketmine\nbt\tag\DoubleTag:68.pocketmine\nbt\tag\DoubleTag:164.5.}.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\DoubleTag:0.pocketmine\nbt\tag\DoubleTag:-0.pocketmine\nbt\tag\DoubleTag:0.}.pocketmine\nbt\tag\ListTag:pocketmine\nbt\tag\ListTag{.pocketmine\nbt\tag\FloatTag:-0.pocketmine\nbt\tag\FloatTag:-0.}.pocketmine\nbt\tag\FloatTag:0.pocketmine\nbt\tag\ShortTag:0.pocketmine\nbt\tag\ShortTag:300.pocketmine\nbt\tag\ByteTag:1.pocketmine\nbt\tag\ByteTag:0.pocketmine\nbt\tag\ShortTag:5.pocketmine\nbt\tag\StringTag:Squid.})
#3 /src/pocketmine/level/Level__64bit(2474): pocketmine\level\format\generic\BaseFullChunk->initChunk(boolean)
#4 /src/pocketmine/level/Level__64bit(2146): pocketmine\level\Level->loadChunk(integer 15, integer 10, boolean 1)
#5 /src/pocketmine/level/Level__64bit(2725): pocketmine\level\Level->getChunk(integer 15, integer 10, boolean 1)
#6 /src/pocketmine/Player__64bit(706): pocketmine\level\Level->populateChunk(integer 15, integer 10)
#7 /src/pocketmine/Player__64bit(1543): pocketmine\Player->sendNextChunk(boolean)
#8 /src/pocketmine/Server__64bit(2412): pocketmine\Player->checkNetwork(boolean)
#9 /src/pocketmine/Server__64bit(2153): pocketmine\Server->tick(boolean)
#10 /src/pocketmine/Server__64bit(2033): pocketmine\Server->tickProcessor(boolean)
#11 /src/pocketmine/Server__64bit(1652): pocketmine\Server->start(boolean)
#12 /src/pocketmine/PocketMine(464): pocketmine\Server->__construct(pocketmine\CompatibleClassLoader object, pocketmine\utils\MainLogger object, string phar://C:\Users\carlf_000\Desktop\PocketMine-MP\PocketMine-MP.phar/, string C:\Users\carlf_000\Desktop\PocketMine-MP\, string C:\Users\carlf_000\Desktop\PocketMine-MP\plugins\)
#13 (1): require_once(string phar://C:/Users/carlf_000/Desktop/PocketMine-MP/PocketMine-MP.phar/src/pocketmine/PocketMine.php)

PocketMine-MP version: 1.6dev #0 [Protocol 45; API 2.0.0]
Git commit: 0000000000000000000000000000000000000000
uname -a: Windows NT MYLAPTOP 10.0 build 10586 (Windows 10) AMD64
PHP Version: 7.0.4RC1
Zend version: 3.0.0
OS : WINNT, win

Loaded plugins:

----------------------REPORT THE DATA BELOW THIS LINE-----------------------

===BEGIN CRASH DUMP===
eNrtWo1v28aS/1f25ABPPoiyKOvDZuEAruO8BnAcX+xc71A9GCtyJbGmSJa7tKwW+d/vN7NLipLt
OH33ChwODoqa3J2dmZ3vGeqPlomXqhX4g+F46Pf6g3GnlUhtzosiK1rBHy2zzrHdSjMTh6rVaS2V
1nJOS2cySYTJhBRLtZyqQszKNDRxlgqtzEf58J8yKVV7X2AhLZMEZ2f48z5O6HC+kEVwcHAWHHzR
qtAHoSyS2W2v1zt4p/SdyfKDqyy8A544Vd7Hq+23Lp0+0EV4kPPyEssHKjWxWR9cxPdxOgdETgQt
sW+CAizBYis46ndappC4ZfBLa68nnjl1zn9ub0eDaWza/fHhfiA2UJOJBZtMrn8r48h7G6exsUfa
0yxLlEz3QXHP/z70g/7wGfQWznt7extmqTZFGZp2Ey5R9yqZTGZZsZRmMlmGhZpDN5PJ2aJM70Q2
/VWFprOFOp0C0Mg5YLJlnpVpdCPnL0P80X0a5CLWBtvBN3efO/wuK6eJouP9wbA7fBFqdPQiiD8a
ANHXv5bb3osQXu87kPwlXL5PMmm+yUIT4utLMM8huV5kxfcBHPaeA/lxbazOXth/kcRzlnNtCvg+
QbCfdr+yV/YfeSW70YH1ooO5SlURhwc/Sq3eI5axK7X9wdE3fTQI4HvSKBcFNFMWTHbb/15d9tVl
X132T7rs4TMue0H/r7P0YDzY8dEdZ3OuDTabvm3Tt3XzZvYefBdRfzB6miiDeW+hlcjijlOj5qig
/GFH1M+9jnA0hc9Uh99FdfyoZNimOlfmzxAdPSJ6lci1Kip64963L5lneZkg+n2LJhMav0DIHw52
Sy0L4L3VKo0u1cMTijp6hPRaFfcbpP2B338GabhQ4d2lMqus2MZ5/BJOf7jLqAXw3qKG3sbl915C
1jv8FrKrIgtRkGfFNlb/Baz+aNh/Bqs2sjDb2B5nxU013h6MBs9geq4ypTwkTYzAeYZOQ1/AC2AF
T2W30sSJnkw+yji9yObzBpRLonUXMZlwGzGZ1H0EgrNtJKDRZu+w82pbiRrhn8P0Tx7Lk3Iep4C1
4j0UbR8iLBSiWqFuszRU7d37/cu6pA0EtUj7rX987bTUX9Lr/bnWC2xYsbSCmUy06rTCLFLE1PgQ
aCZmYuJZ+41ZxBpBWZ8m8T3Rk2kk/u3NStqF/T+AcTxw8BPzJr8TJyJVK2Frr/N7kL+SxEx7/weC
HTZhvbcqjnDAUUGg/BA5uNEOHCEC5CO0QfD5/Prq9OdLPjWuT1mfCIJpAWsP0V87LhyphdTXuVyl
ChUhCFiaR+70V3o55hd6POrhkf76vJSjAonDXX38pGRiFu03comKz7BcjvoOn6MpDWxsWhr1UeZ8
2dNqoV0/BcFP56cXNz/tU4zd6LlCS2weHW44GzjOhk9zJu/V5Y83bctNJdFcFpBeENS7jHS8zWsq
lypCEeC9tRdzWq1KhvakZdcnrU5De04I+xbl0YbPY8vnca/Jp5xq6v7NhmGguARhy9Kx7w71n7wc
NHgBU/40u47nC9O2hiHeWHPHjb++zhZeG5XXRuV1tvA6W3h12VeXfZ0tvM4WXmcLr7OF19nC/+fZ
QqfFoU8m1MdDbxoVB7onvzuK1D34m5ZxErWCHnr/IjNZmAFwMOy0ZB4DrN/tdXuAmscGb73v/IcD
hbxL4imd6R4xhpL6R7z/HKdRttLi8kZ8/O+L06ubT1eIEN2eYEbwODwaiXYFhdghTj++Gw2AgT4n
B60xOBp8PvOx8DvCAlYOHY/Yv800kfhweXmDBX5ZxWmrHmxg4RdIBP0umDEQhn3XbNFdCCBXhYkV
ndu7qt/EWZbO4rmgccqkmKR7P6tIfJSFOBa9cTAcB/DHLzdnot/zR7S/zEx0QioICzkzgbg6F9Zn
aNPS8nIkwxP/2D/s0+JqERvlJagVTrLZjFZkmqKWC5WX23Alw0WMgLtE6alPUDMSJppWeKQ1xf3v
iaMuH9whfdLvMa4kyVbeLKGuuMJvD8s0XspkG+Eym1YLc0hpmUXqhNEglYKfes0hgg1GYVbU7/l9
7k5H8WwWh2Vi1ic+Y2M7NFnhaWUMzF+f0DKnE4+s4wShOIk2a1qpqAFCk7GTd+fvT79c3NCqSiVi
jPdbqYq1I+mWCgSmih967uaIQ0Aenfw7/4NISpN5NO5wB8l8aifqrpcJWYCgwCRCVn4JzmnGQDYg
IAmx7fPpnrhZKK1EdTUhCyXMQokshQGZhTQiJJ0aMVXIjGFSRjCimGZFO7ZHuK6zJU7OCMEuTi1n
qiMy7BSaUIopmo87sc7KwuES8UxAQ/EsZgpQTgEDSdaE+FKtanQHkZpJqEeLVZb+zQiZ5wpGTaJB
yRSHMJs1DRNplmPvvVqoVJT5vJAR/b5jkpI5W2TBJBV7N4tCwYyxBC4Smc5LOVeChogsMmf6OsZ7
kgF//DsLlY6eUTa2t9g+SHMdY3KNMKnS7iq+i3MVxbKbFfMDejugCvs2m91+uP50Ozo89vq3dA5i
rBEFYtJS6XzSIkI/LxTJjgapVFYIGqraaAw1FTLVVMxEdpulyZyCDzwpwypF4IxDJRYyjRJW8hKI
rXdsSPIEFet6URqEMviVHeQSMzYaiDDJtIosW6fko4IiAOcFG63EfSzFf5B9A4Tt3HM7AXgtCf0e
yuqVkGSlGildOCpWVdIhEqWGEUYqhyHw7U6vPgDSLLKI5LTZ8MDWFExt0J+zR1V4aNRLzi5grrAI
YnW6Fs6OAO78r97diGHv5xiCzlKYlIwiJENtLY/YpEIH7EhdFhzgBOxtqkgeEDqypuSx9zy+B+w9
DT9F25FEyN0HcksPfq8orHrQJhUTAXaJ8mXJ03K406lep+GN1HeCij44UJf2kXojts/qkgS1KDIE
YJgEoHVHcGQSLoLBZDsQwxJS05ROWSwrNaW1ZZnCcwiEcV+zyeA+cK4ie0C0NQoiwHXY1NOaNZKq
ZefDjDyUB/3wRJTR5m809y/4VKQo2LMV7pwVbdwAehZ9kgjfwXO3DBgTeSuF7mLNvvr3JJtCsDqb
GWGXYX3L2FBMWqq5nKKd011xbVnpMfFYk3qtp+NOq5gZY2ELSjEWTyV/zXKZoebBg7DjemCZIUZU
FK36cZmSjXae4R4ZuQbJDHTmzKTHjAWiN+FQwVHZINTI6P84+0tw+g3mKXX+r5jXqBCZvgtVNT8O
mxZwKcp9iDKOG6JZsUQ/NbRcXaE3zjgdkS9qIsjtUcVd7W8+SCHURGRjDOHBH5T1NA7jWYoABg+F
3Cg6PCdWK06UPQlHePJ1CzbhhEs44H0bT7bhiCh8ohBCGY7Bu4+B6do2DkUQYa5CSoSaxUK8Whnb
s/UFIEmV6801D3vunnr/SYbcrQ/treeymEL/HmrnxJZi5GFPSpXNiVhxZ4Q7A99dyrTklNvkAuVI
CVvY71QxYccYnDXYuDqbVbFhubGyrvhk8x2/wSIWWZlENSMxIsgqFRvWyW5FzpzjgiNqdFIm9J49
4VFsZFuxp+tLzYpsWYVYOuv2vWZUaugUl3Cq2zWFJ+yncY4q3ZBGAtrK+4KtngIiLxJnNIKh2whb
EXfYPQsUINoCeVQDUC2zwUWoKjU7RzkeOT7fZfbYrvooCVClZI81MVgqDuxfcGdORF6IdkDZWztb
qAjRRp29v4lqkqZ2+FGXbjCNxGbCqTToN6gQ4U+kmoY5wkUldkBbmLpdG4ymqs6JHG72muGr2iHH
L9aGcHeE5zeMmXIfk/VMxQrCynBko8om13IvQIkusuHOKdCexaLjqSt+QrsD8Z6gFobhnl196eAs
OJgi/qziyCxs1Gb/rtF7jD4QY1cYbEoGzYJpgHbFaQR/WshkdoAi31YxkYKdOUIgae0vS5sOWWfn
Bq5GnXT+QM5HZZBMOtaxibBriLsCZqPYtCnXf7lKr6pKYVOzUxFEh1aI9RMq2NPcc6/NmgwdmpqW
c6v+zMVMQdmiYINWDyosbbFDwYe67W69q8kiuIZB292N05jY8F398lb4VLe4JEUFKlOqilM+S2HK
Fa0Tp9XAIrAlpxYH2khTwvYO5qHVEkJkpBv8V23iVguiqnpU2Nmvbb4Y1FkqSY5tx363ibgE5iOe
PRKI+tuMvRD5DJV8KawX9s7xTW7hRxCm5s40yEMg5jE37O5sBXuXn27OA8EBHP+FZUFf4aE/QhQv
84SLYWbPUa7Zq63ldEvxIVoSBCV3VzZIzm7ghAzQkBH2ey4bUVS0SY4skvphWneZzQWQ7eUqHNqy
+po8ny44lVo1iLV9+NwTVDqijw3/qQ1lwu4++79WTS7YHG4IM3VpbqghFGKcpUdVhNUk98ObRpli
CjdUVIhRZJHJSq61xe3wNOxoKxWwGZ3yjysauUS78q9igtgnbNR9IMzTYyAGzx+VBX2as7xbHNsJ
TPjH/SdPs3WlyrakU2WxTdWMghoPbCj4kRpqrHaM04iiNohey3t4N5WKMXXeijq5fO2mDC6t1bUR
x5yZ1IalynJxEXFWJgyWaZiqRy4ahzYrcTdXUGsPwwDaSlS/ZtRUWd/Qckm2suSwS7mqvv+OMkig
lTI+ygchd+WS28m4ijb2sK0LNtLPMopL3Tjm9CAdd+yk9qizO4JH4UMxieZlXplHsMUNg+jalbRE
eGC3SaiW8U2PWPHefcy8i38rGXO7j9a+VE6/7jy7Pa97GsoKxNE/ha36OkPY3DNluR3Ek5R90oP0
iGk7FfTYjjRJkkS5pFk/TW/cqu/CAw3RAmHrxEql9GmhzFE32eLRGiQHD1uzOFzAMqZTjiAhHdLr
ShKhepHX5HIa02+UGMTN72y7H9hgRJOJQuqFsKs2XTsjsdbrmoI6uJFdezQr2l6p08nWKrJcnM6y
jRksMtI9k2x8z+yipmL+0ixdL6ljYA/RkK+uGaUWyG2LzTYzDFuTQs7nyD5uzlBPcKjcpJ8v3T19
G8sPodNP8UPysqbMKt49DZuwu4FVQJF6LjlviklezfLG7OmMxo88D1NJllO26uAdN6Bhn6mqOhQ5
MxTdCsJGfkopzxOQS60laQf+2EAi3GeKRg1RonbVxia4KuA5bKQhu13h1ztCWa1WT4kkiZFurFbO
HyTlW/tCBQvwOyaCihvegrFT0x2IX+jRQ5LpcBf+D8Joo2BVhjQHt3yJqvWw/V81DOfa6qk5MPfJ
nMU0Vwx07SUKiRiMuhTfYJ2pMgP8JGh2Tq7LKWlDLRDvL05vgv4P487w+MHvHD4cdvo/+D9EyMQ2
arVNofCCCHNy1BNz2Le2b4PhPs/J1YNRKQlE0xelM+Sh7a8y0xAFyWJ7jSaoaSSLnVX3W9fGEpng
9sosTmCz22sLOB1/w6LPPjEVR9v7v2r3jWvAECjm1rnZBrm+utheyMPdi3xWM9fbba9rW7PvLBpJ
F4y2V5dr/VvCH6nckxhi0+97sHbh0QejYW/QG+PxzQeoazY8VtPxaDxT/liORuPj4dHxdBj2xqPo
eCaPjqLDaIBs1xNvgNtkdypFBN+Rze9xzlf3D/nuv9uvcA0ALDzw942m2LMlfezr9Q59v3+Mhat3
n7YhrhasPfoe2CfxcX36Xx93hPgI8SqKHh6BUBe0yzaWV0X8SNMoincw5m6QwR/+/O4IS2vJZPlb
5eezMWmDvV3vqOi3BAQO+ZvkmDTQ+vr1fwDktBQq
===END CRASH DUMP===

**Comment by fdfdffdf** [comment]
why?

**Comment by fdfdffdf** [comment]
why have u taken this?

**Comment by PEMapModder** [comment]
@fdfdffdf read.

> @PEMapModder closed this in dd22e49 3 hours ago

Which means, the commit dd22e49 has fixed this issue.
